tools/update_default_release_versions_to_latest.py

```diff
@@ -0,0 +1,388 @@
+    username, _password = get_login(user_login)
+
+    cmd(["rm", "-rf", ASSISTED_SERVICE_CLONE_DIR])
+
+    cmd(["git", "clone",
+         ASSISTED_SERVICE_CLONE_URL.format(user_login=user_login, github_user=username), ASSISTED_SERVICE_CLONE_DIR])
+
+    def git_cmd(*args: str):
+        return cmd(("git", "-C", ASSISTED_SERVICE_CLONE_DIR) + args)
+
+    git_cmd("remote", "add", "upstream", ASSISTED_SERVICE_UPSTREAM_URL)
+    git_cmd("fetch", "upstream")
+    git_cmd("reset", "upstream/master", "--hard")
+
+def commit_and_push_version_update_changes(message_prefix):
+    def git_cmd(*args: str, stdout=None):
+        return cmd(("git", "-C", ASSISTED_SERVICE_CLONE_DIR) + args, stdout=stdout)
+
+    git_cmd("commit", "-a", "-m", f"{message_prefix} Updating versions to latest releases")
+
+    branch = BRANCH_NAME.format(prefix=message_prefix)
+
+    verify_latest_config()
+    status_stdout, _ = git_cmd("status", "--porcelain", stdout=subprocess.PIPE)
+    if len(status_stdout) != 0:
+        for line in status_stdout.splitlines():
+            logging.info(f"Found on-prem change in file {line}")
+
+        git_cmd("commit", "-a", "-m", f"{message_prefix} Updating versions to latest releases")
+
+    git_cmd("push", "origin", f"HEAD:{branch}")
+    return branch
+
+
+def verify_latest_config():
+    try:
+        cmd(["make", "generate-ocp-version"], cwd=ASSISTED_SERVICE_CLONE_DIR)
+    except subprocess.CalledProcessError as e:
+        if e.returncode == 2:
+            # We run the command just for its side-effects, we don't care if it fails
+            return
+        raise
+
+def open_pr(args, task):
+    branch = BRANCH_NAME.format(prefix=task)
+
+    github_client = github.Github(*get_login(args.github_user_password))
+    repo = github_client.get_repo(ASSISTED_SERVICE_GITHUB_REPO)
+    body = " ".join([f"@{user}" for user in PR_MENTION])
+    pr = repo.create_pull(
+        title=PR_MESSAGE.format(task=task),
+        body=body,
+        head=f"{github_client.get_user().login}:{branch}",
+        base="master"
+    )
+    hold_pr(pr)
+    logging.info(f"new PR opened {pr.url}")
+    return pr
+
+
+def hold_pr(pr):
+    pr.create_issue_comment('/hold')
+
+def unhold_pr(pr):
+    pr.create_issue_comment('/unhold')
+
+
+def open_app_interface_pr(fork, branch, current_version, new_version, task):
+    body = " ".join([f"@{user}" for user in PR_MENTION])
+    body = f"{body}\nSee ticket {JIRA_BROWSE_TICKET.format(ticket_id=task)}"
+
+    pr = fork.mergerequests.create({
+        'title': PR_MESSAGE.format(current_version=current_version, target_version=new_version, task=task),
+        'description': body,
+        'source_branch': branch,
+        'target_branch': 'master',
+        'target_project_id': fork.forked_from_project["id"],
+        'source_project_id': fork.id
+    })
+
+    logging.info(f"New PR opened {pr.web_url}")
+    return pr
+
+def get_latest_release_from_major(major_release: str, all_releases: list):
```

Comment:
If you've already renamed the file - please also fix this -  it's been bothering me:

The version terms in the script are wrong. In [OCP terminology](https://docs.openshift.com/container-platform/4.6/release_notes/versioning-policy.html) they use different terms 

e.g. `4.6.10` 
`4` is the major
`6` is the minor
`10` is the ... I don't know what they call it. z-stream, patch, build, micro, whatever. 

But right now the script is calling minor updates "major" and it's pretty confusing